X_Pregrasp:
  wxyz:
  - -0.022167899833769843
  - -0.9993802449948969
  - -0.020349293664945464
  - 0.01826516844263497
  xyz:
  - 0.5084265688306641
  - -0.018498003296773056
  - 0.3632809701034195
